### ISSUE TITLE
Questions on data sets methodology  

### ISSUE THREAD

**Issue by vi-to** [braverock/FactorAnalytics#7]
Hi! First of all, thank you for making your package available!

I am using the `factorDataSetDjia5Yrs` data set you shared to fit some fundamental factors models with `fitFfm()` and I'd like to ask some questions about some variables definitions.

Specifically, on factors, how is `MKTCAP` computed? Is it inflation-adjusted? Then, how is `SIZE` calculated?
On assets returns, it seems `RETURN` is the 'raw' return? What does `RETURN.OLD` stand for?

I checked the manual and ffmVignette, but wasn't able to find descriptions.

**Comment by JustinMShea** [comment]
Hi VIto,

These appear to be part of the CRSP/S&P Global Market data set, and so the definitions should be there accordingly. `SIZE` refers to the Fama-French variable and `MKTCAP` to Market Capitalization. If the authors have the original script used to download this, then we can submit a PR to update their documentation with the complete definitions of each variable.

**Comment by vi-to** [comment]
Hi Justin, thanks for replying.

'Source' in `?factorDataSetDjia5Yrs` was left as TBA, so thank you for indicating the source. (Perhaps worth mentioning it in docs for future reference). From CRSP I found that capitalization has the usual definition (http://www.crsp.org/products/documentation/data-definitions-c#capitalization-end-of-period).

I have a question on `SIZE`. It reportedly holds in general that `SIZE = ln(MKTCAP)` (with ln(.) the natural logarithm), independently of whether `MKTCAP` is inflation-adjusted or computed as of the end-of-month/relative base month (which are nevertheless both aspects worth mentioning in the docs, in my opinion).
However, what I found is:
```r
> cbind(factorDataSetDjia5Yrs$SIZE, log(factorDataSetDjia5Yrs$MKTCAP))
          [,1]     [,2]
 [1,] 10.28838 3.380623
 [2,] 10.89165 3.983894
 [3,] 12.03651 5.128750
 [4,] 10.78167 3.873911
 [5,] 12.08140 5.173647
 [6,] 10.64801 3.740251
 [7,] 12.81826 5.910508
 [8,] 11.60744 4.699681
 [9,] 11.97125 5.063496
[10,] 11.70513 4.797376
[11,] ...
```
What am I doing wrong? Is there some sort of adjustment I'm not considering?
